### PR TITLE
Add version 06

### PR DESCRIPTION
**Pull request by mkazlauskas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ and I would love to add support or fix things that are not working yet**
           cds_version: '#',
           jurisdiction_version: '!',
           postal_code: '33435      ',
-          class: 'I ',
+          klass: 'I ',
```
Comment (winfinit) [inline README.md]: why is this changed from class to klass?

File: index.js [hunk 2]
```diff
@@ -59,7 +68,7 @@
             "cds_version": res3[1],
             "jurisdiction_version": res3[2],
             "postal_code": res3[4],
-            "class": res3[5],
+            "klass": res3[5],
```
Comment (winfinit) [inline index.js:70]: same as in README, why is this changed from "klass" ?

**Comment by mkazlauskas** [comment]
Because class is a keyword and it wouldn't work in my environment. I need to have this so we got to branch it if you want to keep "class"

**Comment by winfinit** [comment]
in what environment "class" key is not accepted?

**Comment by mkazlauskas** [comment]
Dart. ES6 has class keyword as well so there may be potential issues with it too.

**Comment by winfinit** [comment]
don't know about dart, but there is no issue with ES6 having a key called class. I am not feeling "Klass" however if you are ok to change it to something like "dl_class" maybe, to fix a conflict that you are having, then I would be ok with that.

**Comment by mkazlauskas** [comment]
Alright, I think I found a compromise - keep both fields. Having klass alongside class won't hurt anyone, plus I have a feeling there may be other use cases where it's an issue (other transpilers?)
